_posts/2023-06-13-lee23a.md

```diff
@@ -21,7 +21,7 @@ layout: inproceedings
 series: Proceedings of Machine Learning Research
 publisher: PMLR
 issn: 2640-3498
-id: lee23a
+id: cho23a
```

Comment:
```suggestion
id: lee23a
```

Reply:
I think we can't change the simply change the here, as all the links currently point to lee23a.